packages/renderer/src/lib/dashboard/ReleaseNotesBox.svelte

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+import { faCircleArrowUp } from '@fortawesome/free-solid-svg-icons';
+import { Button, CloseButton, Link } from '@podman-desktop/ui-svelte';
+import { onDestroy, onMount } from 'svelte';
+
+import Markdown from '../markdown/Markdown.svelte';
+
+let showBanner = false;
+let notesAvailable = false;
+let updateAvilable = false;
+let notesURL: string;
+let currentVersion: string;
+let title: string = '';
+let summary: string = '';
+let imageUrl: string = '';
+let imageAlt: string = '';
+
+const receiveShowReleaseNotes = window.events?.receive('show-release-notes', () => {
+  showBanner = true;
+});
+
+function openReleaseNotes() {
+  window.podmanDesktopOpenReleaseNotes('current');
+}
+
+function updatePodmanDesktop() {
+  window.updatePodmanDesktop();
+}
+
+async function getInfoFromNotes() {
+  let curVersionSplit = currentVersion.split('.', 2);
+  const urlVersionFormat = curVersionSplit.join('.');
+  notesURL = `https://podman-desktop.io/release-notes/${urlVersionFormat}.json`;
```

Comment:
the frontend component should not have podman-desktop.io,

The link and the fetch should come from the backend part
It'll avoid any CORS issue (and it's not the browser that does the connection, but the backend side)